src/routes.js

```diff
@@ -28,8 +46,12 @@ routes.info = function (req, res, db) {
 }
 
 routes.renderNetwork = function (network, res, db) {
-  db.Pod.projectStats(network, function (projectStats) {
```

Comment:
So is `db.Pod.projectStats` used anywhere after this? Can be removed in `database.js`?